compiler/one-cmds/one-codegen

```diff
@@ -132,6 +164,22 @@ def _parse_arg(parser):
     return codegen_args, backend_args, unknown_args
 
 
+'''
+one-codegen defines its behavior for below cases.
+
+[1] one-codegen -h
+[2] one-codegen -v
+[3] one-codegen -C ${cfg} (backend, command key in cfg)
+[4] one-codegen -C ${cfg} (backends key in cfg)
+[5] one-codegen -b ${backend} ${command}
+[6] one-codegen -b ${backend} -- ${command}
+[7] one-codegen -b ${backend} -C {cfg} (backend, command key in cfg)
+[8] one-codegen -b ${backend} -C {cfg} (backends key in cfg)
```

Comment:
When I was reviewing other codes, I found that [8] has the following behavior.

```suggestion
[8] one-codegen -b ${backend} -C {cfg} (backends key in cfg)
    NOTE Only 'backend' is invoked, even though cfg file has multiple backends.
```

It would be better to add the comment, because it may not be intuitive for some users (or developers). My personal preference is to block using both `-b` and `-C`, but let's go as-is if you think it's good. I leave this comment just want to be careful, because UI is difficult to change after it is released..

Reply:
Yeah, I'll add a comment. Actually, [8] is the main request from https://github.com/Samsung/ONE/issues/10537..

Reply:
(Added)

Now we support multiple backends. If the `-b` and `-C` options cannot be used together, each backend must have multiple cfg files. So, I proposed to introduce a `--backend` option with a `-C` option.